src/mappers/provinces/province_mapper_importer.cpp

```diff
@@ -44,6 +46,42 @@ void VerifyProvinceIsMapped(const mappers::Hoi4ToVic3ProvinceMapping& hoi4_to_vi
    }
 }
 
+std::vector<std::string> AreVic3ProvincesFromSameState(const std::vector<std::string>& provinces_from_map,
+    const std::map<std::string, vic3::StateRegion>& state_regions_from_save)
+{
+   std::map<std::string, std::string> province_to_state_map;
+   for (const auto& state_from_save: state_regions_from_save)
+   {
+      for (const std::string& province_from_map: provinces_from_map)
+      {
+         std::set<std::string> provinces_from_save = state_from_save.second.GetProvinces();
+         if (provinces_from_save.find(province_from_map) != provinces_from_save.end())
+         {
+            province_to_state_map[province_from_map] = state_from_save.first;
+         }
+      }
+   }
+
+   std::set<std::string> state_names;
+   for (const auto& entry: province_to_state_map)
+   {
+      state_names.insert(entry.second);
+   }
+
+   std::vector<std::string> warnings;
+   if (state_names.size() > 1)
+   {
+      for (const auto& province_from_map: provinces_from_map)
+      {
+         std::string warning = "Province " + province_from_map + " is designated as part of " +
```

Comment:
We've been transitioning to using fmt::format for constructing strings. It's the same as std::format, but our linux build machines don't have std::format quite yet.

Reply:
fix with [fmt::format](https://github.com/ParadoxGameConverters/Vic3ToHoI4/pull/632/commits/267089556b4af366e44c321de31c010e4ee0b22d) ?